documentation/home.md

```diff
@@ -0,0 +1,30 @@
+# MuleSoft AI Chain Connector Overview
+
+The MuleSoft AI Chain Connector is designed to help developers easily build and manage AI-driven agents within MuleSoft Anypoint Platform. It provides the tools and support needed to integrate Large Language Models (LLMs), embeddings, and other advanced AI services into MuleSoft applications.
+
+## Key Features
+
+The MuleSoft AI Chain Connector simplifies AI integration into MuleSoft applications with:
+
+- Seamless Interaction with LLMs: Effortlessly integrate and utilize large language models (LLMs) for tasks such as natural language processing, text generation, and more, including advanced features like text generation, analysis, and complex language tasks.
+- Embeddings and Search: Leverage embeddings to handle tasks like text similarity, document search, and clustering, all within MuleSoft applications.
+- Optimized Performance: Designed for high efficiency and performance in enterprise-grade MuleSoft applications, ensuring smooth handling of AI operations.
+- File-Based Local Vector Stores: Create simple, file-based vector stores directly within the connector, ideal for local use or POC designs. For external vector store integration, use the dedicated Vector Store Connector.
```

Comment:
Let's remove mention of the Vector Store connector since it will still be open source at time of launch